lib/credentials.js

```diff
@@ -70,6 +83,12 @@ Credentials.prototype._expand = function() {
     this.requestPubKey = requestDerivation.publicKey.toString();
   }
   var network = WalletUtils.getNetworkFromXPubKey(this.xPubKey);
+  if (typeof this.externalSource == "string") {
```

Comment:
I think using the `hasExternalSource` method below would make code more readable